docs/specs/11_execution_log.md

```
@@ -1,5 +1,13 @@
 # Execution Log 设计规范
 
+> Tracking milestones:
+> - foundational observability reference
+>
+> Related issues:
+> - No dedicated open issue is currently tracked for this spec.
+>
+> Execution source of truth: GitHub milestones and issues.
+
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)